lib/travis/api/v3/services/requests/create.rb

```diff
@@ -13,9 +13,10 @@ def run
 
       user      = find(:user) if access_control.full_access? and params_for? 'user'.freeze
       user    ||= access_control.user
-      remaining = remaining_requests(repository)
+      max       = limit(repository)
+      remaining = remaining_requests(max, repository)
 
-      raise RequestLimitReached, repository: repository if remaining == 0
+      raise RequestLimitReached, repository: repository, max_requests: max, per_seconds: TIME_FRAME.to_i if remaining == 0
```

Comment:
@renee-travisci it is being passed as part of the json body of the HTTP response. so far this only says "request limit reached", but it doesn't give any extra info. this PR adds that extra info

see the test: https://github.com/travis-ci/travis-api/blob/10988118020ea747b9c272894809fd824eeda1a3/spec/v3/services/requests/create_spec.rb#L220-L240